[2022-07-19 10:26:13,301] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: simple_dag.respond scheduled__2022-07-19T10:25:00+00:00 [queued]>
[2022-07-19 10:26:13,316] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: simple_dag.respond scheduled__2022-07-19T10:25:00+00:00 [queued]>
[2022-07-19 10:26:13,316] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-07-19 10:26:13,317] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-07-19 10:26:13,318] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-07-19 10:26:13,341] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): respond> on 2022-07-19 10:25:00+00:00
[2022-07-19 10:26:13,347] {standard_task_runner.py:52} INFO - Started process 7875 to run task
[2022-07-19 10:26:13,352] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'simple_dag', 'respond', 'scheduled__2022-07-19T10:25:00+00:00', '--job-id', '397', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpy6fuq5th', '--error-file', '/tmp/tmpfb2vkxij']
[2022-07-19 10:26:13,354] {standard_task_runner.py:80} INFO - Job 397: Subtask respond
[2022-07-19 10:26:13,443] {task_command.py:370} INFO - Running <TaskInstance: simple_dag.respond scheduled__2022-07-19T10:25:00+00:00 [running]> on host 02ffb48252da
[2022-07-19 10:26:13,598] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=simple_dag
AIRFLOW_CTX_TASK_ID=respond
AIRFLOW_CTX_EXECUTION_DATE=2022-07-19T10:25:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-19T10:25:00+00:00
[2022-07-19 10:26:23,443] {logging_mixin.py:115} INFO - (2724, 1)
[2022-07-19 10:26:23,457] {python.py:173} INFO - Done. Returned value was: Greet Responded Again
[2022-07-19 10:26:23,611] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=simple_dag, task_id=respond, execution_date=20220719T102500, start_date=20220719T102613, end_date=20220719T102623
[2022-07-19 10:26:23,681] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-19 10:26:23,730] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
